gatsby-config.js

```diff
@@ -1,6 +1,9 @@
 const quote = (str) => `"${str}"`;
 
 module.exports = {
+  flags: {
```

Comment:
Is this just a change we are including in this PR that is seperate from the SDK issue? or is this required to fix it? I was just curious.

Reply:
oh, yeah that was just a suggestion to add by Jerel but I can definitely remove it

Reply:
Ya it doesn't seem to work anyways, I think because this branch is using an old version of `gatsby-image` that doesn't support lazy images. With the upgrade to v3, I think this flag is obsolete. Feel free to remove.